### ISSUE TITLE
State of the project

### ISSUE THREAD

**Issue by eliasdaler** [eliasdaler/imgui-sfml#185]
**TL;DR: this project is not actively maintained. I have time for small PR's/issues at the moment. If you want to help out, please contact me.**

Hello, everyone.
As you might have noticed, the updates/work on ImGui-SFML has in the past years significantly gone down. 
Most of it is caused by me not using it in my projects anymore (instead, I'm using SDL and 
imgui_impl_opengl3.cpp  + imgui_impl_sdl.cpp official Dear ImGui backends).

However, I feel like it's my responsibility to maintain this project and not cause it to become obsolete/outdated. 
Thankfully, Dear ImGui doesn't break API compatibility often and any new features which are added can be accessed by just updating Dear ImGui itself - ImGui-SFML doesn't need any work done about it most of the time.

---

That's why I'd be grateful for someone to help me out from time to time. 
If you're actively using ImGui-SFML and want to help maintain it better, I might give you write permissions to this repo so that it won't take long to get PRs approved. However, you need to do one of the following:

1. You previously made a non-trivial PR for the repo which got approved by me
2. You made some PR's to Dear ImGui or SFML which got merged

If you didn't do any of these things, you'd need to go through standard PR procedure here and afterwards you'd get a write permission.

Please e-mail me at eliasdaler@protonmail.com or DM me on Twitter (@eliasdaler) to get into contact with me if you want to get a write permission for the project to move/fix things faster.

---

Thanks a lot for the patience, everyone. I know that this project moves too slowly at times, but that's how things go when you stop using them yourself. :)
